packages/server/graphql/mutations/deleteUser.ts

```diff
@@ -74,20 +74,14 @@ export default {
       })
     }
     // do this after 30 seconds so any segment API calls can still get the email
+    const update = {
+      isRemoved: true,
+      email: 'DELETED',
```

Comment:
yes, we should definitely have unique constraint on the email field. however: i came to the conclusion (and thought you agreed, but it may have been miscommunicated) that using pg constraints upfront made the equality checking part of the migration process more complicated than it needs to be. i think we should leave pg constraints off until we've verified that both tables are equal.

i also like tabling the constraint part of it until later, because it forces us to be more deliberate about why we want to migrate to postgres. it seems like the primary objective is stability and performance, and better data integrity is also very important but secondary.

Reply:
ahhh, i was thinking we were only treating foreign keys as strings until both tables were ready in pg. 
sure, let's the email field for the next PR, but definitely should get it in before we remove the write to rethinkdb.